docs/source/api_reference/transforms.rst

```diff
@@ -65,7 +65,8 @@ Embedding transforms and their utilities:
 
    EmbeddingSegmentTransform
    EmbeddingWindowTransform
-   embeddings.TS2VecEmbeddingModel
```

Comment:
Why have we made this change?

Reply:
I think we should emphasize that it is model + we import them from `embeddings.models` everywhere in code